tox.ini

```diff
@@ -24,7 +24,7 @@ deps =
     pytest-flakes
     pytest-pep8
     pytest-travis-fold
-    sanic
+    sanic==20.9.1
```

Comment:
I am not going to merge this with a 20.9 dependency.

Just add this line to the top of `conftest.py` after the imports.

```python
Sanic.test_mode = True
```

Reply:
In the coming weeks, we will release: https://github.com/sanic-org/sanic-test

It will automatically set that.